api/src/test/java/org/pmiops/workbench/actionaudit/ActionAuditServiceTest.kt

```diff
@@ -49,7 +49,7 @@ class ActionAuditServiceTest {
 
     @Test
     fun testSendsSingleEvent() {
-        actionAuditService!!.send(EVENT_1!!)
+        actionAuditService!!.send(EVENT_1)
```

Comment:
Why this change, and what does it do?

Reply:
It is, and it's also on other branches.

It fixes a Kotlin warning about unnecessary promotion of a nullable type to a non-nullable type. Initially the argument was `event1: ActionAuditEvent?`, in order to allow instantiating it in the `@Before` method. When I changed it to a constant without the question mark, that made the compiler enforce a non-null constraint on it everywhere, which makes the `!!` in this call redundant.